File: ThirtyDaysOfCodeInPython/conditionals_3.py
import sys
# Conditional Statements - https://www.hackerrank.com/challenges/30-conditional-statements?h_r=next-challenge&h_v=zen

# Given an integer, , perform the following conditional actions:
# If  is odd, print Weird
# If  is even and in the inclusive range of  to , print Not Weird
# If  is even and in the inclusive range of  to , print Weird
# If  is even and greater than , print Not Weird 

scan = sys.stdin
n = int(scan.readline())
def checkInteger(n):
	ans = "Weird"

	if (n % 2 != 0):
		print ans
		return
	elif (2<=n<=5):
		ans = "Not Weird"
	elif (n>20):
		ans = "Not Weird"
	print ans

checkInteger(n)

File: ThirtyDaysOfCodeInJava/conditionals_3.java
/*
Conditional Statements - https://www.hackerrank.com/challenges/30-conditional-statements?h_r=next-challenge&h_v=zen
*/

/*
Given an integer, , perform the following conditional actions:

If  is odd, print Weird
If  is even and in the inclusive range of  to , print Not Weird
If  is even and in the inclusive range of  to , print Weird
If  is even and greater than , print Not Weird
*/
class Main{
	public static void main(String[] args) {
	      Scanner scan = new Scanner(System.in);
	      int n = scan.nextInt(); 
	      scan.close();
	       
	      checkInteger(n);

	   }
	  
		public static void checkInteger(int n){
			String ans ="Weird";
			
	        if(n % 2 != 0){
	            
	        }else if(2 <= n && n <= 5)
				ans = "Not Weird";
			else if(n > 20)
				ans = "Not Weird";

			System.out.println(ans);
		}
}